data-scripts/lists.ts

```diff
@@ -225,6 +225,24 @@ export default {
       },
     },
   },
+  'ja': {
+    commonWords: {
+      // fixme We must convert Japanese characters (Hiragana, Katanaka and Kanji) into Romaji as a dictionary
+      // because Japanese characters are not permitted to use for password in general.
+      // It would be too difficult to do programmatically.
```

Comment:
Why can't the japanese writing system be used as a password? Is it really not permitted to have them in a password?  You could have emojis and other unicode characters, too.
A password shouldn't care about the unicode characters because it is encrypted anyway.

Reply:
@MrWook Thank you for your review.

When we type japanese characters, [Input method editor](https://en.wikipedia.org/wiki/Input_method) which is called IME in general is used. 

e.g. 
Following steps are the way to type "花", which means "flower" in English, written by Kanji.

- Type "はな" which also means "flower" but written by Hiragana
- Convert "はな" into "花" by using IME

![image](https://i.gyazo.com/fb0b6c068597165f7a76d4520f65f0c6.gif)

The serious problem when the japanese characters are used for password is that `<input type="password">` field does not support IME conversion. In other words, the IME couldn't be launched when we are typing.

This is a screenshot when I tried to type "はな" and convert it into "花" as well in password field.
The IME didn't work at all.

![image](https://i.gyazo.com/3f99d23db93e46887289a5d36f378252.gif)

It could be a solution for this that only hiragana letters which do not need IME to type are supported for password but I seems to be weird for japanese people so that it became common for us no to support all japanese characters in password.